src/Network/DogStatsd.hs

```diff
@@ -0,0 +1,55 @@
+module Network.DogStatsd (
+ dogStatsdClient
+, increment
+, decrement
+, count
+, gauge
+, timing
+, histogram
+) where
+
+import Network.Statsd (Stat, Type(..), fmtDatagram)
+import Network.Statsd.UdpClient(UdpClient, fromURI, send)
+
+import Control.Monad
+import Data.Maybe
+import Data.List
+import Data.Time.Units
+import Text.Printf
+import Network.URI
+
+type Tags = [(String, String)]
```

Comment:
> Can you replace these String types with the new names below

Good idea, done in c94e0ac0a8f8efdcdd8c718fe5db8eb669bfde44

> Do we need to add these types to the export list at the top too

I don't think so. I'm consuming the library from another project and exporting those doesn't seem to be required.







> Edit: one last thing I thought of, I’m not sure what the Haskell convention would be specifying `type Tags = [(Name, Value)]` vs `type Tag = (Name, Value)` and pushing the list wrapper down to the function definitions, what do you think?

My thought is that Haskell does this for things like `String`, so what I have is probably fine, but I'm happy to code it either way:
``` hs
> :i String
type String = [Char] 	-- Defined in ‘GHC.Base’
```